app/src/main/java/org/oppia/android/app/settings/profile/ProfileListActivity.kt

```diff
@@ -23,6 +25,10 @@ class ProfileListActivity : InjectableAppCompatActivity() {
     return false
   }
 
+  override fun routeToProfileEditActivity(profileId: Int) {
+    startActivity(ProfileEditActivity.createProfileEditActivity(this, profileId, false))
```

Comment:
add inline comments so that it becomes more readable

Reply:
I'm sorry I'm unable to understand this

Reply:
I think Rajat might be referring to using either argument comments or named arguments (the latter being preferred if it's a Kotlin method), i.e.

``ProfileEditActivity.createProfileEditActivity(arg1 = this, profileId, arg2 = false)``

Reply:
done